src/Command/CreateFamilyCommand.php

```diff
@@ -70,6 +70,9 @@ protected function configure()
                 ),
                 self::YES
             )
+            ->addOption('category-options', null, InputOption::VALUE_OPTIONAL,
+                sprintf('Create %s field as a "multiple_options" attribute with these options (comma-separated)', self::CATEGORIES),
```

Comment:
```suggestion
                sprintf('Create %s field as a "multiple_options" attribute with these options (comma-separated) instead of text attributes', self::CATEGORIES),
```